src/repo/index.js

```diff
@@ -0,0 +1,96 @@
+const githubAPI = require('./github')
+const gitlabAPI = require('./gitlab')
+
+const SUPPORTED_REPO_TYPES = {
+  github: {
+    value: 'github',
+    name: 'GitHub',
+    checkKey: 'login',
+    defaultHost: 'https://github.com',
+    linkToCommits: '<%= options.repoHost %>/<%= options.projectOwner %>/<%= options.projectName %>/commits?author=<%= contributor.login %>',
+    linkToIssues: '<%= options.repoHost %>/<%= options.projectOwner %>/<%= options.projectName %>/issues?q=author%3A<%= contributor.login %>',
```

Comment:
Could we add `|| "https://github.com"` here like we have for gitlab? It makes sense to me. Could you open up another PR for this @tigermarques?

Reply:
Actually, don't bother. I'll go ahead and do it really quick :)